File: Java/백준/Bronze/22015. 金平糖 （Konpeito）/金平糖 （Konpeito）.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int[] a = new int[3];
        for (int i =0; i<3 ;i++){
            a[i] = in.nextInt();
        }
        int sum=0;
        int max = a[0]>a[1]?a[0]:a[1];
        max = max>a[2]?max:a[2];
        for (int i=0; i<3; i++){
            sum+=max-a[i];
        }
        System.out.println(sum);
    }
}

File: C#/백준/Bronze/22015. 金平糖 （Konpeito）/金平糖 （Konpeito）.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa=Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int sum=0;
            for (int i=0; i<3; i++){
                sum+=aa.Max()-aa[i];
            }
            Console.WriteLine(sum);
        }
    }
}